cmd/machine-config-daemon/pivot.go

```diff
@@ -38,57 +35,15 @@ func init() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 }
 
-func run(_ *cobra.Command, args []string) (retErr error) {
-	flag.Set("logtostderr", "true")
-	flag.Parse()
-
-	var container string
-	if fromEtcPullSpec || len(args) == 0 {
-		fromEtcPullSpec = true
-		data, err := os.ReadFile(etcPivotFile)
-		if err != nil {
-			if os.IsNotExist(err) {
-				return fmt.Errorf("no container specified")
-			}
-			return fmt.Errorf("failed to read from %s: %w", etcPivotFile, err)
-		}
-		container = strings.TrimSpace(string(data))
-	} else {
-		container = args[0]
-	}
-
-	client := daemon.NewNodeUpdaterClient()
-
-	osImageContentDir, err := daemon.ExtractOSImage(container)
-	if err != nil {
-		return err
-	}
-	changed, err := client.Rebase(container, osImageContentDir)
-	if err != nil {
-		return err
-	}
-
-	// Delete the file now that we successfully rebased
-	if fromEtcPullSpec {
-		if err := os.Remove(etcPivotFile); err != nil {
-			if !os.IsNotExist(err) {
-				return fmt.Errorf("failed to delete %s: %w", etcPivotFile, err)
-			}
-		}
-	}
-
-	if !changed {
-		glog.Info("No changes; already at target oscontainer")
-	}
-
-	return nil
-}
-
 // Execute runs the command
 func Execute(cmd *cobra.Command, args []string) {
-	err := run(cmd, args)
-	if err != nil {
-		fmt.Printf("error: %v\n", err)
-		os.Exit(1)
-	}
+	fmt.Println(`
+	ERROR: pivot no longer forces a system upgrade. It will be fully depreceated in a later y release. 
```

Comment:
`depreceated` -> `deprecated`, but maybe we should just say `removed` instead, since we are technically already deprecating it

Reply:
Makes sense to me, updated to `removed`